packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -163,16 +163,10 @@ class MultiSelectList extends React.Component {
       newProps.isSelected = isSelected;
     }
 
```

Comment:
That is a good point. Fixed [here](https://github.com/cerner/terra-core/pull/294/commits/df5e2565500f08672ccad1c1bbc518d47a0888c8#diff-0e18bc4abc1b29779e0a97b763acd271R180).